juriscraper/opinions/united_states/federal_special/uscg_ccap.py

```diff
@@ -0,0 +1,46 @@
+# Scraper for US Coast Guard Court of Criminal Appeals
+
+from juriscraper.lib.string_utils import convert_date_string
+from juriscraper.OpinionSite import OpinionSite
+from requests.utils import requote_uri
+
+
+class Site(OpinionSite):
+    def __init__(self, *args, **kwargs):
+        super(Site, self).__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.current_page = 1
+        self.base_url = (
+            "https://www.uscg.mil/Resources/Legal/"
+            "Court-of-Criminal-Appeals/"
+            "CGCCA-Opinions/smdpage15701/{}"
+        )
+
+    def __format_url(self):
```

Comment:
am I missing something?  Does this get called somewhere? Is this related to python's magic `__format__` method?

Reply:
That is 100% something I forgot to clean up with a typo in the method name. I will clean that up.

Reply:
I cleaned it up. Leaving ` _format_url()` in (with a correct name) to make implementing `_download_backwards()` easy if it's needed.

Reply:
Ok, now I see the method being used, so it makes sense.  

Just FYI, `_download_backwards` will not run on this scraper.  The default behavior is nothing happens when back-scraping a site (see [here](https://github.com/freelawproject/juriscraper/blob/master/juriscraper/AbstractSite.py#L415-L417)).  

You can test by running the following from within the repo root on your terminal:

```
python sample_caller.py -c juriscraper.opinions.united_states.federal_special.uscg_ccap --backscrape
```

So, if you want a back scraper to be available, you will need to override and implement `_download_backwards`, and will also need to implement the `self.back_scrape_iterable` property in the `__init__`.  The GA scraper is a decent example: https://github.com/freelawproject/juriscraper/blob/master/juriscraper/opinions/united_states/state/ga.py

Reply:
Thanks, I'll probably do that later this week, just wanted to dip a toe in haha.